src/features/pastelConf/__tests__/pastelConf.test.ts

```diff
@@ -46,9 +45,17 @@ describe('pastelConf/updateDefaultPastelConfig', () => {
     })
     jest.spyOn(fs.promises, 'readFile').mockResolvedValueOnce(mResponse)
 
-    const result = await updateDefaultPastelConfig('somePath')
+    await updateDefaultPastelConfig('somePath')
     expect(fs.promises.writeFile).toBeCalledWith('somePath', mResponse)
+  })
 
-    expect(result).toBe(1)
+  test('catches an error if pastel.conf is undefined', async () => {
+    try {
```

Comment:
For tests that assert in catch block, we need `expect.hasAssertions` because we want this test to throw an error that will go into catch block. `expect.hasAssertions` will fail the test if an error is not thrown

Reply:
I have updated for this